README.md

```diff
@@ -1,19 +1,40 @@
-
 # ImageLoaderPlugin ![Build Status](https://github.com/ManiVaultStudio/ImageLoaderPlugin/actions/workflows/build.yml/badge.svg?branch=master)
 Image loader plugin for the [ManiVault](https://github.com/ManiVaultStudio/core) framework.
 
 ```bash
 git clone https://github.com/ManiVaultStudio/ImageLoaderPlugin.git
 ```
 
+<p align="middle">
+  <img src="https://github.com/ManiVaultStudio/ImageLoaderPlugin/assets/58806453/bca56677-847a-493a-a1c2-980f2130c136" align="middle"  width="80%" />
+</p>
+
+## Features 
+- Overview of image information for all image files in a given directory:
+  - Size, number of points (pixels), channels and more
+  - Filter images by name
+- Edit meta data like data set and dimension names
+- Select which dimensions are to be loaded
+- Multiple image formats: PNG, JPEG, BPM and (single page) TIF/TIFF
+  - RGB images
+  - "High-dimensional" images, i.e. more than three values per pixel. All grayscale (single channel) image files in a folder are interpreted as part of one single image, e.g. the files in `./data/images/stack/sun/`
+- Data layout interpretation:
+  - Image sequence (default): Each grayscale image in a folder is interpreted as an image channel, i.e., given 100 images of 28x28 pixels this loader creates a dataset of 28x28=784 data points with 100 dimensions (values per pixel). The image dimensions is 28x28 pixel.
```

Comment:
The image stack layout is the default, I would describe that one first:
- Image stack (default): In this data layout, same-sized images are layered on top of each other. Pixels with the same coordinate in each layer together form a data point, where each element represents a dimension. Given 100 images of 28x28 pixels, this loader creates a dataset of 28x28=784 data points with 100 dimensions (values per pixel). The image dimensions is 28x28 pixels.
- Image sequence: In this case, each image of a same-size sequence of images is considered a data point and each pixel is considered a dimension. Given 100 images of 28x28 pixels, this loader creates a dataset of 100 data points with 28x28=784 dimensions.

Reply:
Ah, I got them mixed up! I fixed that and expanded the explanation